tests/python/test_transformer_engine.py

```diff
@@ -23,6 +23,21 @@ class ComputeType(Enum):
     BACKWARD = auto()
 
 
+@pytest.fixture(scope="module")
+def setup_process_group(mpi_test) -> None:
+    os.environ["MASTER_ADDR"] = "localhost"
+    # The default port as used by https://github.com/pytorch/pytorch/blob/45a8b5682eb69d865cbf68c7f2f689b56b4efd53/torch/csrc/distributed/c10d/TCPStore.hpp#L51.
+    os.environ["MASTER_PORT"] = "29500"
+    dist.init_process_group(
+        backend="nccl",
+        init_method="env://",
```

Comment:
Address and Port should be read from env variables, like in: https://github.com/NVIDIA/Fuser/blob/d9cee3852dffab13bed7e4e1034f42971c5e2b0f/csrc/multidevice/communicator.cpp#L102

in particular, setting address as "localhost" excludes the possibility of running the test on a multi-node setup.

Btw, if, for some reason, you still want to use `localhost:29500`, then instead of overwrite env vars you can do:

```suggestion
    # The default port as used by https://github.com/pytorch/pytorch/blob/45a8b5682eb69d865cbf68c7f2f689b56b4efd53/torch/csrc/distributed/c10d/TCPStore.hpp#L51.
    dist.init_process_group(
        backend="nccl",
        init_method="tcp://localhost:29500",
```

Reply:
TIL!